recipes/default.rb

```diff
@@ -772,6 +772,19 @@
   end
 end
 
+if node['hopsworks']['audit_log_dump_enabled'].eql? "true"
```

Comment:
It's better if you use `casecmp` which makes case-insensitive comparison

Reply:
do you mean 
if node['hopsworks']['audit_log_dump_enabled'].casecmp? "true"

Reply:
or
node['hopsworks']['audit_log_dump_enabled'].casecmp("true") == 0